5_databases/week3/creatingdatabase.sql

```diff
@@ -0,0 +1,66 @@
+-- Active: 1699100471406@@0.0.0.0@3306
+--> Database Creating
+
+CREATE DATABASE mealsharing;
+
+USE mealsharing;
+
+CREATE TABLE meals (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255),
+    description TEXT,
+    location VARCHAR(255),
+    meal_when DATETIME,
+    max_reservations INT,
+    price DECIMAL(10,2),
+    created_date DATE
+);
+
+CREATE TABLE reservations (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    number_of_guests INT,
+    meal_id INT,
+    created_date DATE,
+    contact_phonenumber VARCHAR(255),
+    contact_name VARCHAR(255),
+    contact_email VARCHAR(255),
+    FOREIGN KEY (meal_id) REFERENCES meals(id) ON DELETE CASCADE
+);
+
+CREATE TABLE reviews (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255),
+    description TEXT,
+    meal_id INT,
+    stars INT,
+    created_date DATE,
+    FOREIGN KEY (meal_id) REFERENCES meals(id) ON DELETE CASCADE
+);
```

Comment:
Make sure to also pay attention to which columns should be NULLABLE or not. If omitted, the column will be made NULL. However, it is common that certain data need to be present for the data to be consider valid. 

I.e. how can a reservation be useful if we don't know which meal it is for or who placed the reservation?